src/main/java/com/example/tour/place/PlaceDTO.java

```diff
@@ -15,4 +20,8 @@
     private double longitude;
     @JsonProperty(value= "description")
     private String description;
+    @JsonProperty(value= "images")
+    private List<Image> images;
```

Comment:
Shouldn't this be private List<ImageDTO>?

Reply:
it is a private list of images